apps/module-scan/src/store.test.ts

```diff
@@ -170,6 +170,25 @@ test('batch cleanup works correctly', async () => {
   const batches = await store.batchStatus()
   expect(batches).toHaveLength(1)
   expect(batches[0].id).toEqual(firstBatchId)
+  expect(batches[0].label).toEqual('Batch 1')
+
+  const thirdBatchId = await store.addBatch()
+  await store.addBatch()
```

Comment:
Unnecessary? Removing this would allow you to ditch the `expect.arrayContaining` below, I think.

Reply:
This batch gets deleted, I was trying to test that but I shouldn't be using arrayContaining I was just trying to ignore the order of it but I'll update to just sort the array first.